core/utils.js

```diff
@@ -37,28 +38,23 @@ const userAgent = goog.require('Blockly.utils.userAgent');
 
 
 /**
- * Don't do anything for this event, just halt propagation.
+ * Halts the propagation of the event without doing anything else.
  * @param {!Event} e An event.
  */
 const noEvent = function(e) {
+  deprecation.warn(
+    'Blockly.utils.noEvent', 'September 2021', 'September 2022');
   // This event has been handled.  No need to bubble up to the document.
   e.preventDefault();
   e.stopPropagation();
 };
 exports.noEvent = noEvent;
 
-/**
- * Is this event targeting a text input widget?
- * @param {!Event} e An event.
- * @return {boolean} True if text input.
- */
 const isTargetInput = function(e) {
-  return e.target.type == 'textarea' || e.target.type == 'text' ||
-      e.target.type == 'number' || e.target.type == 'email' ||
-      e.target.type == 'password' || e.target.type == 'search' ||
-      e.target.type == 'tel' || e.target.type == 'url' ||
-      e.target.isContentEditable ||
-      (e.target.dataset && e.target.dataset.isTextInput == 'true');
+  deprecation.warn(
+    'Blockly.utils.isTargetInput', 'September 2021', 'September 2022',
+    'Blockly.browserEvets.isTargetInput');
+  return browserEvents.isTargetInput;
```

Comment:
Doesn't this need parens?:

```suggestion
  return browserEvents.isTargetInput();
```